scale_info/overrides/overrides.ts

```diff
@@ -0,0 +1,36 @@
+import { Codec } from "../../deps/scale.ts"
+import * as $ from "../../deps/scale.ts"
+import { Ty } from "../raw/Ty.ts"
+import { ChainError } from "./ChainError.ts"
+import { $era } from "./Era.ts"
+
+export const overrides: Record<string, (ty: Ty, visit: (i: number) => Codec<any>) => Codec<any>> = {
+  Option: (ty, visit) => {
+    return $.option(visit(ty.params[0]!.ty!))
+  },
+  Result: (ty, visit) => {
+    return $.result(
+      visit(ty.params[0]!.ty!),
+      $.instance(
+        ChainError,
+        $.tuple(visit(ty.params[1]!.ty!)),
+        ChainError.toArgs,
+      ),
+    )
+  },
+  BTreeMap: (ty, visit) => {
+    return $.map(visit(ty.params[0]!.ty!), visit(ty.params[1]!.ty!))
+  },
+  BTreeSet: (ty, visit) => {
+    return $.set(visit(ty.params[0]!.ty!))
+  },
+  "frame_support::traits::misc::WrapperOpaque": (ty, visit) => {
```

Comment:
Nit: would we want a `fmtPath` util from `scale_info`, which we could use like so:

```ts
fmtPath("frame_support", "traits", "misc, "WrapperOpaque")
```

Reply:
This `::` format is only used here, so I don't think this is worthwhile.

I don't think
```ts
[fmtPath("frame_support", "traits", "misc", "WrapperOpaque")]: ...
```
is cleaner than
```ts
"frame_support::traits::misc::WrapperOpaque": ...
```
anyway.